stdlib/Pkg/src/REPLMode.jl

```diff
@@ -1246,7 +1246,6 @@ Create a project called `pkgname` in the current folder.
     precompile
 
 Precompile all the dependencies of the project by running `import` on all of them in a new process.
-The `startup.jl` file is disabled during precompilation unless julia is started with `--startup-file=yes`.
```

Comment:
?

Reply:
Precompile function got changed, this is no longer applicable.